iac/terraform/aws/web-client/variables.tf

```diff
@@ -0,0 +1,20 @@
+variable "environment" {
+  type        = string
+  description = "Deployment environment"
```

Comment:
i believe we are using this as a prefix for our resources, let's update description accordingly

Reply:
Though I copied from another previous code. Changing there as well then